.github/actions/update-snapshots/action.yml

```
@@ -26,7 +26,7 @@ inputs:
   browser:
     description: The playwright browser to install (one of [`chromium`, `firefox`, `webkit`])
     required: false
-    default: chromium
+    default: ''
```

Should we keep chromium as default?

```suggestion
    default: chromium
```

---

Makes sense.